src/dotnet-svcutil/lib/tests/Baselines/OptionsRoundtrip/SvcutilOptions/dotnet-svcutil.svcutil.json

```diff
@@ -1,6 +1,6 @@
 {
   "providerId": "Microsoft.Tools.ServiceModel.Svcutil",
-  "version": "99.99.99.1313",
+  "version": "99.99.99",
```

Comment:
Is this change necessary?

Reply:
The original baseline saved only replaced partial version number of a result output, with the change applied, we get clear and common match in baseline comparison for the generated result.